tests/test_requests.py

```diff
@@ -285,6 +286,24 @@ def test_request_cookies():
     assert response.text == "Hello, cookies!"
 
 
+def test_request_url_for():
+    app = Starlette()
+
+    @app.route("/users/{name}")
+    async def func_users(request):  # pragma: no cover
+        ...
```

Comment:
Instead of "pragma: no cover" consider "raise NotImplementedError()", which could be automatically ignored then (not sure if that was taken already though.. :/)

Reply:
Didn't know about that one… had to adapt the coverage config (`tests/.ignore_lifespan`).